lib/list_all.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+class ListAll extends StatelessWidget {
+  const ListAll({super.key, required this.listOfTasks, required this.title});
+  final Set<String> listOfTasks;
```

Comment:
Nit: add a blank line after the constructor.

```suggestion

  final Set<String> listOfTasks;
```